snuba/migrations/migration.py

```diff
@@ -51,3 +53,12 @@ def forwards_local(self) -> Sequence[Operation]:
     @abstractmethod
     def backwards_local(self) -> Sequence[Operation]:
```

Comment:
if the execution method is in the migration, it seem you could avoid exposing the sequences of steps and make them protected instead.

Reply:
Sure, renamed